config/application.rb

```diff
@@ -24,7 +24,25 @@
 module Dradis
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 7.0
+    config.load_defaults 7.2
+
+    # ** Please read carefully, this must be configured in config/application.rb **
```

Comment:
Should we add a `TODO` here to uncomment the line once the relevant Dradis version is released?

Reply:
This line can actually be uncommented 👍 updated